### PR TITLE
Updated README | Better repo/build structure

### PR DESCRIPTION
**Pull request by abejgonzalez**
Addresses comments in #1. Reorganizes the repo into different directories according to the tools used. Includes a new build file and README update.

### CODE REVIEW

File: base/Dockerfile [hunk 1]
```diff
@@ -1,9 +1,9 @@
 FROM openjdk:11.0.1-jdk
 
-# man directory is missing in some base images
+# Man directory is missing in some base images
 # https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=863199
 RUN apt-get update \
-    && mkdir -p /usr/share/man/man1 \
+    && mkdir -p /usr/share/man/man1 \
```
Comment (ucbjrl) [inline base/Dockerfile]: This looks like a typo - the backslash is quoting a space?

**Review by ucbjrl — CHANGES_REQUESTED** [review]
Please verify that the EOL quoting in base/Dockerfile is correct - (in GitHub diff, it looks wrong). Otherwise, looks good.

File: base/Dockerfile (hunk 1 above)
Comment (abejgonzalez) [reply base/Dockerfile]: No. The backslash is not quoting a space, it is extending the command to the next line. I double checked the docker build and this does not affect it. However, I went ahead and removed all trailing whitespace.

**Review by abejgonzalez — COMMENTED** [review]
Went ahead and removed all whitespace at EOLs.

**Review by ucbjrl — APPROVED** [review]

**Pull request merged**